sql/parser/builtins.go

```diff
@@ -294,7 +294,7 @@ var builtins = map[string][]builtin{
 
 	"translate": {stringBuiltin3(func(s, from, to string) (Datum, error) {
 		const deletionRune = utf8.MaxRune + 1
-		translation := make(map[rune]rune, len(from))
+		translation := make(map[rune]rune, utf8.RuneCountInString(from))
```

Comment:
The size parameter for maps is just a hint as to the number of elements that will be added to it. I can't imagine this change making any difference it practice.